connector_infor_account_move/models/account_move.py

```diff
@@ -0,0 +1,301 @@
+        backends = self.env['infor.backend'].search([])
+        for backend in backends:
+            sync_journal_ids = backend.mapped('infor_journal_ids.odoo_id')
+            for move in self:
+                if move.journal_id not in sync_journal_ids:
+                    continue
+                values = self._infor_prepare_binding(backend)
+                self.env['infor.account.move'].create(values)
+
+
+class InforAccountMove(models.Model):
+    _name = 'infor.account.move'
+    _inherit = 'infor.binding'
+    _inherits = {'account.move': 'odoo_id'}
+    _description = 'Infor Account Move'
+
+    odoo_id = fields.Many2one(
+        comodel_name='account.move',
+        string='Move',
+        required=True,
+        ondelete='cascade',
+    )
+    # When not empty, indicate that we already generated an infor message for
+    # this record so we will no longer export it. At some point, the infor
+    # message will be dropped, and this field set to null, but we should have
+    # an infor id returned by infor (in external_id)
+    infor_message_id = fields.Many2one(
+        comodel_name='infor.message',
+        string='Infor Message',
+        ondelete='set null',
+        index=True,
+    )
+    infor_journal_id = fields.Many2one(
+        comodel_name='infor.account.journal',
+        compute='_compute_infor_journal_id',
+        store=True,
+        index=True,
+    )
+
+    @api.depends('backend_id', 'odoo_id.journal_id')
+    def _compute_infor_journal_id(self):
+        for binding in self:
+            infor_journal_model = self.env['infor.account.journal']
+            binding.infor_journal_id = infor_journal_model.search(
+                [('backend_id', '=', self.backend_id.id),
+                 ('odoo_id', '=', self.odoo_id.journal_id.id),
+                 ],
+                limit=1
+            )
+
+    def is_realtime(self):
+        return self.infor_journal_id.is_realtime
+
+
+class InforMoveListener(Component):
+    _name = 'infor.account.move.listener'
+    _inherit = 'base.connector.listener'
+    _apply_on = ['infor.account.move']
+
+    @skip_if(lambda self, record, **kwargs: self.no_connector_export(record))
+    def on_record_create(self, record, fields=None):
+        if record.is_realtime():
+            record.with_delay().generate_message()
+
+
+class InforMoveProducer(Component):
+    _name = 'infor.account.move.producer'
+    _inherit = 'infor.jinja.producer'
+    _apply_on = ['infor.account.move']
+
+    _template_path = 'connector_infor_account_move/messages/move.xml.tmpl'
+
+    @staticmethod
+    def _format_numeric(n):
+        """Format monetary value."""
+        return '{0:.5f}'.format(n)
+
+    @staticmethod
+    def _default_text(moves):
+        """Default description for some field."""
+        dates = set(moves.mapped('date'))
+        if len(dates) == 1:
+            date = next(iter(dates))
+            return 'STJ-{}'.format(''.join(date.split('-')))
```

Comment:
@TDu It should be dynamic to support other journals, not hard-coded.

Reply:
Oops sorry I misunderstood, fixing